test/src/main/java/org/apache/accumulo/test/fate/FlakyFate.java

```diff
@@ -39,11 +39,6 @@ public class FlakyFate<T> extends Fate<T> {
   public FlakyFate(T environment, FateStore<T> store, Function<Repo<T>,String> toLogStrFunc,
       AccumuloConfiguration conf) {
     super(environment, store, false, toLogStrFunc, conf, new ScheduledThreadPoolExecutor(2));
-  }
-
-  @Override
-  protected void startFateExecutors(T environment, AccumuloConfiguration conf,
-      Set<FateExecutor<T>> fateExecutors) {
     for (var poolConfig : getPoolConfigurations(conf).entrySet()) {
       fateExecutors.add(
           new FlakyFateExecutor<>(this, environment, poolConfig.getKey(), poolConfig.getValue()));
```

Comment:
Is any of this still needed? I think it can be removed if the FatePoolWatcher runs for FlakyFate

Reply:
If we remove this, then `FlakyFateExecutor`'s won't be used, right? The FatePoolWatcher adds `FateExecutor` instances.